core/src/main/java/org/apache/brooklyn/location/ssh/SshMachineLocation.java

```diff
@@ -1104,4 +1013,36 @@ public String findPassword() {
         return getConfig(SshTool.PROP_PASSWORD);
     }
 
+    /** @deprecated since 5.0.1; mutex-related methods are now accessible via {@link #mutexes()} */
```

Comment:
Should be `since 1.0.0` (same elsewhere).

Reply:
D'oh!